scripts/commands/godmode.lua

```diff
@@ -1,16 +1,17 @@
 -----------------------------------
 -- func: godmode
 -- desc: Toggles god mode on the player, granting them several special abilities.
+-- Pass variable of 1 to command to enable a "soft" god mode.
 -----------------------------------
 
 cmdprops =
 {
     permission = 1,
-    parameters = ""
+    parameters = "i"
 }
 
-function onTrigger(player)
-    if (player:getCharVar("GodMode") == 0) then
+function onTrigger(player, arg1)
+    if (not arg1 or arg1 == 0) and player:getCharVar("GodMode") == 0 then
```

Comment:
You only need to handle arg1 once at the top of the function, rather than in both blocks:

```lua
local mode = utils.clamp(arg1 or 0, 0, 2)
local state = player:getCharVar("GodMode")

...
```